packages/lib/modules/lbp/LbpPreview.tsx

```diff
@@ -53,49 +67,63 @@ export function LbpPreview() {
         }}
       >
         <VStack align="start" p="lg" spacing="lg" w="full">
-          <Flex w="full">
-            <Heading color="font.maxContrast" size="md">
-              LBP Preview
-            </Heading>
+          {!isLastStep && (
+            <>
+              <Flex w="full">
+                <Heading color="font.maxContrast" size="md">
+                  LBP Preview
+                </Heading>
 
-            <Spacer />
+                <Spacer />
 
-            <Button
-              _hover={{ color: 'font.linkHover' }}
-              color="font.link"
-              position="relative"
-              top="4px"
-              variant="ghost"
-              onClick={onOpen}
-            >
-              Get help
-            </Button>
-          </Flex>
+                <Button
+                  _hover={{ color: 'font.linkHover' }}
+                  color="font.link"
+                  position="relative"
+                  top="4px"
+                  variant="ghost"
+                  onClick={onOpen}
+                >
+                  Get help
+                </Button>
+              </Flex>
 
-          <TokenSummary
-            chain={chain}
-            projectInfoForm={projectInfoForm}
-            launchTokenMetadata={launchTokenMetadata}
-          />
+              <TokenSummary
+                chain={chain}
+                projectInfoForm={projectInfoForm}
+                launchTokenMetadata={launchTokenMetadata}
+              />
+            </>
+          )}
+
+          <HStack w="full">
+            <SimpleInfoCard
+              title={`${launchTokenMetadata.symbol} start price`}
+              info={`$${fNum('fiat', maxPrice)}`}
+            />
+            <SimpleInfoCard title="Sale market cap" info={saleMarketCap} />
+            <SimpleInfoCard title="FDV market cap" info={fdvMarketCap} />
+          </HStack>
 
           <PoolWeights
-            startTime={startTime}
-            endTime={endTime}
+            startTime={saleStructureData.startTime}
+            endTime={saleStructureData.endTime}
             startWeight={startWeight}
             endWeight={endWeight}
             launchTokenMetadata={launchTokenMetadata}
             collateralToken={collateralToken}
```

Comment:
could reference the obj directly for `collateralToken` and delete ln 31

Reply:
changed.